volumetric/volAddBorder.cpp

```diff
@@ -102,6 +102,7 @@ int main(int argc, char**argv)
   general_opt.add_options()
     ( "help,h", "display this message." )
     ( "input,i", po::value<std::string>(), "Input vol file." )
+    ( "inside", "Add border inside the volume (set 0 inside the border)." )
```

Comment:
Update the option doc as well ?

Reply:
yes oops..

Reply:
I also update the doxygen description that I forget ;)